src/fs.js

```diff
@@ -24,19 +24,28 @@ function updateFile(filename, content, exists) {
 
 /** Copy files from a given directory to the current working directory */
 function copyFiles(sourceDir, files, options) {
+	options = options || { overwrite: true };
```

Comment:
This default value will be applied only when no options are passed, which is confusing. 

```js
function copyFiles(sourceDir, files, options = {}) {
   const { overwrite  = true, errorOnExist } = options;
```